filesystems/localfilesystem/src/main/kotlin/com/lightteam/localfilesystem/repository/LocalFilesystem.kt

```diff
@@ -210,8 +233,20 @@ class LocalFilesystem(private val defaultLocation: File) : Filesystem {
         return Single.create { emitter ->
             val file = File(fileModel.path)
             if (file.exists()) {
-                val text = file.readText(fileParams.charset)
-                emitter.onSuccess(text)
+                val charset = if (fileParams.chardet) {
+                    CJKCharsetDetector.DEBUG = BuildConfig.DEBUG
```

Comment:
What does it do?

Reply:
> What does it do?

It's just a switch to print the log and see the encoding information that is automatically detected